client/client.go

```diff
@@ -6,8 +6,7 @@ import (
 )
 
 const (
-	statusOK    = "OK"
-	statusERROR = "ERROR"
+	statusOK = "OK"
```

Comment:
Just checking, we don't need `statusERROR` anymore?

Reply:
It is never used, so figured we might as well remove it